smqtk_relevancy/impls/libsvm_hik.py

```diff
@@ -92,13 +94,13 @@ def __init__(self, descr_cache_filepath=None, autoneg_select_ratio=1,
         self.cores = cores
 
         # Descriptor elements in this index
-        self._descr_cache = []
+        self._descr_cache = []  # type: List
```

Comment:
The type annotation can be a non-commend: `self._descr_cache: List = []`. Might also want to annotate the expected contents of the list.